packages/components/src/Actions/ActionSplitDropdown/__snapshots__/ActionSplitDropdown.test.js.snap

```diff
@@ -16,6 +17,7 @@ exports[`ActionSplitDropdown should render "no option" item when items array is
       </span>
     </span>
   }
+  toggleLabel="Open \\"{{label}}\\" menu"
```

Comment:
not sure if it is intended or is a jest snapshot generation bug

Reply:
No this is du to the mock we provide that just returns the default value